epregressions/diffs/table_diff.py

```diff
@@ -468,7 +468,10 @@ def table_diff(thresh_dict, inputfile1, inputfile2, abs_diff_file, rel_diff_file
                 # Statistics local to this table
                 for diff_result in diff_dict[h]:
                     diff_type = diff_result[2]
-                    if diff_type == 'small':
+                    if h == 'Version ID':
```

Comment:
Special case here, if the table heading is "Version ID", then the diff calculation is ignored, and it is considered an equal comparison.